src/main/scala/caravan/bus/common/DummyMemory.scala

```diff
@@ -0,0 +1,55 @@
+package caravan.bus.common
+import chisel3._
+import chisel3.experimental.ChiselEnum
+import chisel3.stage.ChiselStage
+import chisel3.util.{Cat, Decoupled, MuxLookup}
+import chisel3.util.experimental.loadMemoryFromFile
+
+
+//implicit parameters for Config, Request and Response
+class DummyMemController(programFile: Option[String])(implicit val config: BusConfig, implicit val request: AbstrRequest, implicit val response: AbstrResponse) extends Module {
```

Comment:
Need to remove the `programFile` functionality since it would not work with masked memories correctly

Reply:
I am commenting it from everywhere.

Maybe in future, we can find a way to `loadMemoryFromFile` with masked memories as well.

Reply:
Disabling `ProgramFile` is affecting some test cases..